client/extensions/wp-job-manager/components/navigation/index.jsx

```diff
@@ -58,10 +60,22 @@ class Navigation extends Component {
 	}
 
 	render() {
+		const { siteSlug, translate } = this.props;
+
 		return (
-			<SectionNav selectedText="Settings">
-				<SectionNavTabs>{ values( Tabs ).map( tab => this.renderTabItem( tab ) ) }</SectionNavTabs>
-			</SectionNav>
+			<div>
+				<HeaderCake
+					backText={ translate( 'Plugin Overview' ) }
+					backHref={ siteSlug && addSiteFragment( '/plugins/wp-job-manager', siteSlug ) }
+				>
+					WP Job Manager
+				</HeaderCake>
+				<SectionNav selectedText="Settings">
+					<SectionNavTabs>
+						{ values( Tabs ).map( tab => this.renderTabItem( tab ) ) }
```

Comment:
Could be shortened as `{ values( Tabs ).map( this.renderTabItem ) }`

Reply:
That would unfortunately lose the arrow's implicit `this` binding, so it'd need to be

```js
{ values( Tabs ).map( this.renderTabItem, this ) }
```

([see](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#Parameters)). To which I honestly prefer sticking with the arrow.

Thanks for reviewing tho! :slightly_smiling_face: